src/main/java/net/minecraftforge/fmllegacy/server/ServerLifecycleHooks.java

```diff
@@ -97,7 +100,11 @@ public static boolean handleServerAboutToStart(final MinecraftServer server)
 
     public static boolean handleServerStarting(final MinecraftServer server)
     {
-        DistExecutor.runWhenOn(Dist.DEDICATED_SERVER, ()->()->LanguageHook.loadLanguagesOnServer(server));
+        DistExecutor.runWhenOn(Dist.DEDICATED_SERVER, ()->()->{
+            LanguageHook.loadLanguagesOnServer(server);
+            if (!(server instanceof GameTestServer))
```

Comment:
This could probably use a comment at the end to explain this. I understand why it is happening but to someone in the future reading the code it probably looks rather backwards.